apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -22,16 +23,34 @@ const styles = {
     flexGrow: 1,
     padding: 10,
     borderLeft: 'solid 1px black'
+  },
+  nav: {
+    margin: '10px 0px',
+    fontSize: 18
+  },
+  navLink: {
+    color: color.purple,
+    margin: '0px 5px'
   }
 };
 
 class LessonOverview extends Component {
   static propTypes = {
-    displayName: PropTypes.string.isRequired,
-    overview: PropTypes.string,
+    lesson: PropTypes.shape({
+      course: PropTypes.shape({
```

Comment:
should lesson, course and unit be required? it looks like course and unit are depended upon below. I would kind of expect that a lesson might not have a unit group but I'm not clear if there would still be a course link in that case.

Reply:
Thank you for pointing this out. I had definitely not thought through this case. I'm going to follow up with Tess as I also wonder if we just want to point back to the script page and not have both course and script since it is getting very long.

Reply:
Removed the course link based on conversation with Tess